services/datamanager/builtin/file_deletion.go

```diff
@@ -62,7 +65,8 @@ func exceedsDeletionThreshold(ctx context.Context, captureDirPath string, fsSize
 			}
 			dirSize += fileInfo.Size()
 			if float64(dirSize)/fsSize >= captureDirToFSUsageRatio {
-				logger.Warnf("At threshold to delete, going to delete usage ratio %f threshold %d", float64(dirSize)/fsSize, captureDirToFSUsageRatio)
+				logger.Warnf("Disk is close to full, current datacapture dir disk usage(%f) exceeds threshold(%f)",
```

Comment:
[nit] To make this messaging user-friendly, can be helpful to have this explicitly say "current disk usage of the data capture directory (%f) exceeds threshold (%f)"

Reply:
done